sunpycube/cube/cube_utils.py

```diff
@@ -519,16 +517,38 @@ def get_cube_from_sequence(cubesequence, item):
     """
     if isinstance(item, int):
         result = cubesequence.data[item]
-
     if isinstance(item, slice):
-        data = cubesequence.data[item]
-        meta = cubesequence.meta
-        result = cubesequence._new_instance(data, meta)
-
+        data = cubesequence.data[item.start:item.stop]
```

Comment:
I see you indexed with just the slice below.  Is there a reason you haven't done that here too?